build_projects/update-dependencies/Program.cs

```diff
@@ -60,6 +60,20 @@ public static void Main(string[] args)
             }
         }
 
+        private static BuildInfo GetBuildInfo(string name, string buildInfoFragment, bool fetchLatestReleaseFile = true)
```

Comment:
It seems odd that `fetchLatestReleaseFile` is defaulted to 'true', but it is only called in 2 places, both passing in 'false'.